inst/rmd/qc_report.rmd

```diff
@@ -420,7 +423,87 @@ if(has_processed){
 
 Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
 The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
-Here we show the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+The above plot shows the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+The below plots show the same UMAP embeddings, coloring each cell by the gene expression of the gene indicated in the label for each panel. The genes chosen for plotting are the top 12 highly variable genes identified in the library.
+
+```{r message=FALSE}
+if(has_processed){
+  
+  if(has_var_genes){
```

Comment:
I think we only need `if(has_var_genes){` here? 

In the current setup, if `has_var_genes` is False, the `ggplot` will fail as `coldata_df` wasn't yet set up, correct?

For this to work, we will want to initialize `has_var_genes <- FALSE` before line 387, or make the line something like: 
``` 
if(has_processed & has_var_genes){
```